### ISSUE TITLE
Create script to discover users directly granted access to resources

### ISSUE THREAD

**Issue by NickCorcoran** [bcgov/cloud-pathfinder#2885]
**Describe the Issue**
Some user accounts in GCP do not show up in our Google Workplace tenancy, but are instead granted direct access to resources with their unmanaged account.

**Additional Context**
Add any other context, attachments or screenshots

**Acceptance Criteria**
- [x] Create script to discover user accounts directly granted access to resources
- [x] Run script in GCP

Once this is complete, a separate task will be created to communicate to those users and get them migrated into our managed user pool.
